pkg/util/executor/interface.go

```diff
@@ -16,8 +20,20 @@ func NewLocalExecutor(envVars []string) *LocalExecutor {
 	}
 }
 
+const (
+	localExecutorAppend = "/host"
```

Comment:
This should probably be called something like `localExecutorPrefix` (vs. "append", that being adding to the end).